src/TestLogger/Core/ITestResultStore.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Spekt Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.TestLogger.Core
+{
+    public interface ITestResultStore
```

Comment:
Needs `IEnumerable<TestResultInfo> GetResults();` or similar.

Reply:
Absolutely needed. Will add in the future PR where we start using the store implementation.